src/util/sexpr/format.h

```diff
@@ -302,7 +302,6 @@ bool get_pp_unicode(options const & o);
 format pp(name const & n);
 
 /** \brief Format a S-expression */
-format pp(sexpr const & s, options const & o);
 format pp(sexpr const & s);
```

Comment:
why did you have to remove it?

Reply:
This one was just a "free" cleanup. This function is not defined anywhere; this declaration was left behind and VS flags declarations without definitions.